GETTING-STARTED.md

```diff
@@ -0,0 +1,110 @@
+# Getting Started
+
+## Setup and Installation
+
+### Prerequisites
+
+In order to use **Django Deploy**, you must first install the following dependencies:
+- [Python](https://www.python.org/downloads/) 3.5 or higher
```

Comment:
Should we tell users to check with python --version ?

Reply:
I added a note in the installation section but I'm not super happy with how I did it...

Reply:
i think its clear on our github about 3.5, at this point if that fails for them then GG, else we can add another check to make sure their python and fail fasT?